codebuild/bin/apache2/sites-enabled/renegotiate.conf

```diff
@@ -1,4 +1,4 @@
-<VirtualHost *:11111>
+<VirtualHost *:7777>
```

Comment:
I'm not familiar with apache-- does this need to match the ports.conf? If so, why do we set the port in two different places?

Reply:
The virtual host port restricts the scope of the configuration to just one port, so it should match the port we tell apache to listen to in ports.conf. I made this a variable so we only set it once in ports.conf.